src/contexts/SessionContext.js

```diff
@@ -0,0 +1,17 @@
+import React, { createContext, useState } from 'react';
+
+export const SessionContext = createContext({
+    /*eslint-disable-next-line*/
+    setSessionID: (sessionID) => {},
+    sessionID: null,
+});
+
+export function SessionContextProvider({ children }) {
+    const [sessionID, setSessionID] = useState(null);
```

Comment:
Don't be afraid of `undefined` - just handle it properly :)
```suggestion
    const [sessionID, setSessionID] = useState();
```